README.rst

```diff
@@ -32,11 +33,10 @@ Credentials *(passwords set at first boot)*
 -------------------------------------------
 
 -  Webmin, SSH: username **root**
--  postgresql, phppgadmin: username **postgres**
+-  postgresql: username **postgres**
 
 
 .. _PostgreSQL: http://www.postgresql.org/
 .. _comparison: http://www.wikivs.com/wiki/MySQL_vs_PostgreSQL
 .. _TurnKey Core: http://www.turnkeylinux.org/core
-.. _PHPPgAdmin: http://phppgadmin.sourceforge.net/
```

Comment:
Instead of just deleting, you need to change this to:
`.. _Adminer: http://adminer.org/`
Otherwise the Adminer link (where Adminer is mentioned in the features) doesn't work. (I.e. that's how links are done in restructured text)